### PR TITLE
added TestFile entry to PackageInfo.g

### PR DESCRIPTION
**Pull request by mohamed-barakat**

### CODE REVIEW

File: ExamplesForHomalg/PackageInfo.g [hunk 1]
```diff
@@ -103,7 +103,7 @@ AvailabilityTest := function()
     return true;
   end,
 
-
+TestFile := "maketest.g",
```
Comment (fingolfin) [inline ExamplesForHomalg/PackageInfo.g:106]: Does this file exist?

Comment (mohamed-barakat) [reply ExamplesForHomalg/PackageInfo.g:106]: Yes, it is in the repository.

Comment (mohamed-barakat) [reply ExamplesForHomalg/PackageInfo.g:106]: Ah, you probably mean `4ti2Interface/PackageInfo.g`. Correct, fixed.

Comment (mohamed-barakat) [reply ExamplesForHomalg/PackageInfo.g:106]: Did this resolve your question?

Comment (fingolfin) [reply ExamplesForHomalg/PackageInfo.g:106]: yes

**Pull request merged**